spec/feature/queue/case_details_spec.rb

```diff
@@ -1799,4 +1799,66 @@
       end
     end
   end
+
+  describe "Unscheduled hearing notes" do
+    let!(:current_user) do
+      user = create(:user, css_id: "BVASYELLOW", roles: ["Build HearSched"])
+      User.authenticate!(user: user)
+    end
+    let(:fill_in_notes) { "Fill in notes" }
+
+    before do
+      HearingsManagement.singleton.add_user(current_user)
+    end
+
+    shared_examples "edit unscheduled notes" do
+      it "edits unscheduled successully" do
+        id = appeal.is_a?(Appeal) ? appeal.uuid : appeal.vacols_id
+
+        visit("/queue/appeals/#{id}")
+
+        within('div#hearing-details') do
+          expect(page).to have_content(COPY::UNSCHEDULED_HEARING_TITLE)
+          expect(page).to have_content("Type: #{appeal.readable_current_hearing_request_type}")
+          click_button("Edit", exact: true)
+          fill_in "Notes", with: fill_in_notes
+          click_button("Save", exact: true)
+          expect(page).to have_content(fill_in_notes)
+          expect(page).to have_content("Last updated by BVASYELLOW on 01/01/2020")
```

Comment:
```suggestion
          expect(page).to have_content("Last updated by BVASYELLOW on #{Time.zone.now.strftime('%m/%d/%Y')}")
```
in case someone ever changes the frozen time